neo/test/iotest/test_asciisignalio.py

```diff
@@ -133,8 +134,8 @@ def reader(filename, comment_rows):
 
         os.remove(filename)
 
-    # test usecols
-    # test skiprows
+    # test_skiprows
+    # test_usecols
```

Comment:
I don't think this change is related to the NWBIO change you suggest. In that case can you remove it here?

Reply:
Yes. I had a mix of several of my branches. It is done. I removed it.